### DISCUSSION TITLE
textbox height has no effect when sets value below 30

### DISCUSSION THREAD

**Discussion by buskysnow** [AvaloniaUI/Avalonia#14911] [Q&A]
### Describe the bug

when i tried to changed the textbox's height to a value less than 30, i didn't see any changes.
but if i sets it to a bigger value , the  height of textbox changed.

### To Reproduce

	<TextBox Height="20" FontSize="12" Width="199"></TextBox>

### Avalonia version

11.0.10

### OS

Windows

**Comment by timunie** [comment]
Set MinHeight as well to a lower value. Some controls have min sizeses.

**Comment by buskysnow** [comment]
> Set MinHeight as well to a lower value. Some controls have min sizeses.



> Set MinHeight as well to a lower value. Some controls have min sizeses.

thanks for your reply. that solved my problem . it works well now.
but the Combo Box control has the same problem. and set MinHeight to it has no effect too.
so what i should do?

**Reply by timunie** [reply]
1. Use DevTools F12 to inspect visual tree (maybe ComboBoxItem is limiting element here)
2. Check out `<FluentTheme DensityStyle="Compact" />` in case you use FluentTheme
3. Check original styles: https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Themes.Fluent/Controls/ComboBox.xaml
